config/config_example_avr_legacy_standardplus_mode.h

```diff
@@ -0,0 +1,18 @@
+/* Configuration example
+
+This example is targetted at the AVR platform (Arduino Uno & friends), only!
+
+The configuration formerly known as STANDARD_PLUS is still the default on AVR, so you
+do not need to configure anything! This file just lists the relevant settings involved: */
+
+#include "MozziConfigValues.h"  // for named option values
```

Comment:
If I understand well, this is actually not needed in any example right?

Reply:
Strictly it is not needed, but again - after single compilation unit - a user may want to put ```#include <some_config_example.h>``` near the top of their sketch (and before ```#include <MozziGuts.h>```). Then the config value labels need to be known at that point, and thus MozziConfigValues.h is included.